keps/prod-readiness/sig-node/2238.yaml

```diff
@@ -3,3 +3,5 @@ alpha:
   approver: "@johnbelamaric"
 beta:
   approver: "@johnbelamaric"
+stable:
+  approver: "@wojtek-t"
```

Comment:
Please don't change approver in the middle of the feature - if John approved the previous phases, he should also approve the stable one.